### ISSUE TITLE
Primary primaryFieldName

### ISSUE THREAD

**Issue by DominicWatts** [ashsmith/magento2-blog-module-tutorial#23]
Great tutorial.  One of the best.

Is primaryFieldName correct?

https://github.com/ashsmith/magento2-blog-module-tutorial/blob/master/view/adminhtml/ui_component/blog_post_listing.xml#L28

**Comment by JanisE** [comment]
Why does it work then?

**Comment by DominicWatts** [comment]
No idea. Although grids have evolved further. This is not latest spec.

**Comment by ashsmith** [comment]
Yep, I need to investigate this as I noticed it wasn't the latest whilst working on a similar project recently.

I will be starting a new post soon that will break down the admin ui components in more detail, so I'll also be updating my previous posts to bring it up to date (has it really been 2 years!?) with best practises, and to line up better with Magento 2.2 and any other changes.
